processing/data_collection/gazette/pipelines.py

```diff
@@ -22,6 +22,21 @@ def process_item(self, item, spider):
         return item
 
 
+class DefaultValuesPipeline:
+    """ Add defaults values field, if not already set in the item """
+
+    default_field_values = {
+        "territory_id": lambda item, spider: getattr(spider, "TERRITORY_ID"),
+        "scraped_at": lambda item, spider: dt.datetime.utcnow(),
+    }
+
+    def process_item(self, item, spider):
+        for field in self.default_values:
+            if field not in item:
+                item[field] = self.default_field_values(field)(item, spider)
```

Comment:
**issue**: `self.default_field_values` is a `dict`. So it is not callable. I believe that `self.default_field_values[field](item, spider)` will solve that.

**question**: why are you passing `item` to the functions? It is not used by any of them.

Reply:
You're right about both issues. They were due a (bad) last minute refactor. My fault. :facepalm: 
Regarding the `item`, I added it "just in case" someone wants to add another default value that used it. But, yes it doesn't make sense right now. Let me remove it for the sake of YAGNI :)